### DISCUSSION TITLE
How is nextjs.org/docs made?

### DISCUSSION THREAD

**Discussion by TheMikeyRoss** [vercel/next.js#55724] [Help] [answered]
### Summary

Hi,

I'm currently in a deep dive trying to figure out how next.js docs is made.
I know the mdx text content is in [here](https://github.com/vercel/next.js/tree/canary/docs) but I'm trying to figure out how the 
non text template design is made. Especially the "Was this helpful?" button at the bottom of each page 

I'm not sure if the docs are made using nextra or not. And I can't find the source code for the "was this helpful?" button.

Any thoughts?

**Comment by rohanharikr** [comment, marked as answer]
Looks like the source code powering nextjs.org/docs is currently closed source per https://github.com/vercel/next.js/discussions/54936
